src/views/anon/intro/VentureDirectory.tsx

```diff
@@ -0,0 +1,231 @@
+import { VBSelect, VBTextField } from '../../../components/VBForms'
+import { initializeOrganizations } from '../../../store/organizationReducer'
+import { AppDispatch } from '../../../store/store'
+
+import ventureDirectory from '../../../assets/venture_directory.svg'
+import { industriesList } from '../../../utils/industries'
+import { VentureCard } from '../../../components/VBCards'
+import { ASCEND, sortByKey, sortList } from '../../../utils/sort'
+
+export interface IVentureStats {
+    title: string,
+    value: number,
+    text: string
+}
+
+const VentureStats = ({title, value, text} : IVentureStats) => (
+    <Box>
+        <Typography marginBottom='10px' marginTop='20px' variant='h3' color='#DC6803'>{value}+</Typography>
+        <Typography marginBottom='10px' variant='body1' fontWeight='bold'>{title}</Typography>
+        <Typography marginBottom='10px' variant='body1'>{text}</Typography>
+    </Box>
+)
+
+const searchId = 'org-search'
+
+const VentureDirectory = () => {
+    const organizations = useAppSelector(state => state.organizations)
+    const dispatch = useAppDispatch()
+    const [searchParams] = useSearchParams()
+    const navigate = useNavigate()
+    const [orgQuery, setOrgQuery] = useState('')
+    const [industryQuery, setIndustryQuery] = useState('')
+    const [sortBy, setSortBy] = useState(ASCEND)
+    const [organizationsToDisplay, setOrganizations] = useState(organizations.data)
+
+    const stats: Array<IVentureStats> = [
+        {
+            title: 'Companies created',
+            value: 120,
+            text: 'We\'ve helped build over 120 amazing startup ventures.'
+        },
+        {
+            title: 'Jobs created',
+            value: 360,
+            text: 'Our ventures have generated more than 360 employment opportunities.'
+        },
+        {
+            title: 'Venture capital investors',
+            value: 210,
+            text: 'Our ventures have garnered the interest of over 210 VC investors.'
+        },
+        {
+            title: '5-star reviews',
+            value: 150,
+            text: 'We\'re proud of our 5-star rating backed by 150+ founders\' reviews.'
+        }
+    ]
+
+    useEffect(() => {
+        const orgSearch = searchParams.get(organizationQueryPrefix)
+        const industrySearch = searchParams.get(industryQueryPrefix)
+        const sortValue = searchParams.get(sortByKey)
+
+        if (orgSearch || industrySearch || sortValue)
+            updateDisplay(orgSearch, industrySearch, sortValue)
+        else
+            initializeSearch(dispatch)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [searchParams, dispatch])
+
+    useEffect(() => {
+        if (organizationsToDisplay.length == 0)
+            setOrganizations(organizations.data)
+    }, [organizations, organizationsToDisplay])
+
+    const updateDisplay = async (org: string | null, industry: string | null, sortBy: string | null) => {
```

Comment:
left search to backend but sort to frontend, not sure if this is the best decision maybe we will move sort to backend later

Reply:
For MVP, I'm good with sorting on the frontend. We're getting a list of all organizations anyway, we can think about adding more querying to the list of organizations later.